src/test/java/net/snowflake/client/jdbc/HeartbeatIT.java

```diff
@@ -85,7 +78,7 @@ public static void tearDownClass() throws Exception {
    * @param queryIdx The query index
    * @throws SQLException Will be thrown if any of the driver calls fail
    */
-  private void submitQuery(boolean useKeepAliveSession, int queryIdx)
+  void submitQuery(boolean useKeepAliveSession, int queryIdx)
```

Comment:
curious why remove `private`?

Reply:
This was so that the submitQuery() function in the subclass HeartbeatAsynchronousLatestIT.java could override this submitQuery() function. It could probably be protected rather than package-private, however.